src/geometry/PolySetBuilder.cc

```diff
@@ -60,7 +60,8 @@ int PolySetBuilder::vertexIndex(const Vector3d &pt)
 
 void PolySetBuilder::appendPoly(const std::vector<int> &inds)
 {
-  ps->indices.push_back(inds);        
+  auto& face = ps->indices.emplace_back();
+  face.insert(face.begin(), inds.begin(), inds.end());
```

Comment:
This is probably food for another round of refactoring, but I've been wondering if we should start using spans for use-cases such as this. `std::span` is available since C++20, but the compatible `boost::span` has been in Boost since 1.78 (December 2021). Not sure if we can require that quite yet, but could be worth figuring out.

Reply:
It's more a question of whether all target distros have boost-1.78 available..